docs/docs/3-features-modules/9-brevo-module/migration-guide/migration-from-brevo-v2-to-v3.md

```diff
@@ -115,6 +115,20 @@ Instead define them in the `BrevoConfigProvider` once:
 </BrevoConfigProvider>
 ```
 
+### Add Brevo test contacts page to admin interface
+
+Create `BrevoTestContactsPage` with `createBrevoTestContactsPage` from `@comet/brevo-admin` and add it to your project's `MasterMenu`:
+
+```tsx
+const BrevoTestContactsPage = createBrevoTestContactsPage({
+    scopeParts: ["domain", "language"],
```

Comment:
We can remove that because it's not necessary anymore from [v3.0.3](https://github.com/vivid-planet/comet-brevo-module/releases/tag/v3.0.3)

```suggestion
```